edit-post/components/header/index.js

```diff
@@ -50,14 +51,17 @@ function Header( {
 						forceIsDirty={ hasActiveMetaboxes }
 						forceIsSaving={ isSaving }
 					/>
-					<IconButton
-						icon="admin-generic"
-						onClick={ toggleGeneralSidebar }
-						isToggled={ isEditorSidebarOpened }
-						label={ __( 'Settings' ) }
-						aria-expanded={ isEditorSidebarOpened }
-					/>
-					<MoreMenu key="more-menu" />
+					<PinnedPlugins>
```

Comment:
Question is if we should also convert `Document Settings` into a plugin. I put this fill in here as a temporary solution.

Reply:
Out of scope for this PR.